readme.md

```diff
@@ -54,10 +54,10 @@ Compilation
 Follow the instructions at the [megasource][megasource] repository page.
 
 ### *nix
-Run `platform/unix/automagic` from the repository root, then run ./configure and make.
+Pick a directory to store generated files to, say `build/`:
 
-	$ platform/unix/automagic
-	$ ./configure
+	$ cmake -B build  # this will create the directory `build/`
+	$ cd build/
 	$ make
```

Comment:
I suggest not to invoke `make` directly as this is not portable.
```suggestion
	$ cmake -B build -S. --install-prefix $PWD/prefix # this will create the directory `build/`
	$ cmake --build build --target install # this will put the files in `prefix/`.
```